### PR TITLE
Added `showEdgeCutStatistics` option

### PR DESCRIPTION
**Pull request by ravil-mobile**
Hi all, 

This PR adds an option to display the edge-cut statistics. I personally need it for one experiment. If you think it can be a useful feature then we may consider adding it to the master.

What do you think?

### CODE REVIEW

**Comment by Thomas-Ulrich** [comment]
Interesting for comparing different mesh or clustering strategies?
Might be difficult to interpret the value, don't you think?

File: src/Initializer/InitProcedure/InitMesh.cpp [hunk 1]
```diff
@@ -175,5 +186,14 @@ void seissol::initializer::initprocedure::initMesh() {
   // Call the post mesh initialization hook
   seissol::Modules::callHook<seissol::POST_MESH>();
 
-  logInfo(seissol::MPI::mpi.rank()) << "End init mesh.";
+  logInfo(commRank) << "End init mesh.";
+
+  if ((seissolParams.mesh.showEdgeCutStatistics) && (commSize > 0)) {
```
Comment (krenzland) [inline src/Initializer/InitProcedure/InitMesh.cpp:191]: Shouldn't this be > 1 or am I missing something?

**Review by krenzland — CHANGES_REQUESTED** [review]
Looks great, just one small question. (Not sure if it's a bug or whether I am missing something)

File: src/Initializer/InitProcedure/InitMesh.cpp (hunk 1 above)
Comment (ravil-mobile) [reply src/Initializer/InitProcedure/InitMesh.cpp:191]: You're right

Comment (ravil-mobile) [reply src/Initializer/InitProcedure/InitMesh.cpp]: done

**Pull request merged**